courseSchedule.py

```diff
@@ -0,0 +1,26 @@
+class Solution(object):
+    def canFinish(self, numCourses, prerequisites):
+        """
+        :type numCourses: int
+        :type prerequisites: List[List[int]]
+        :rtype: bool
+        """
+        summary = {}
+        queue = []
+        degree = [0] * numCourses
+        for pair in prerequisites:
+            summary[pair[1]] = summary.get(pair[1], []) + [pair[0]]
```

Comment:
Check my comments in "Q210 Add courseScheduleII.py"

Reply:
Sure